src/svelte/src/components/dataEditor.svelte

```diff
@@ -1662,10 +1684,24 @@ limitations under the License.
     display: flex;
   }
 
+  .dataEditor div.measure.selection {
+    flex-direction: column;
+    text-align: center;
+    justify-content: center;
```

Comment:
Super minor, but I think ergonomically, speaking the selection text and clear button should be left justified and not centered because it's easier to get to that clear button if it's at a fixed location. Centering it makes it move around.

Reply:
Just pushed an update. Take a look and let me know what you think.